e2e-tests/IIoTPlatform-E2E-Tests/Discovery/DiscoveryTestTheory.cs

```diff
@@ -32,14 +33,14 @@ public class DiscoveryTestTheory {
         }
 
         [Fact, PriorityOrder(0)]
-        public void Test_Discover_All_OPC_UA_Endpoints() {
+        public async Task Test_Discover_All_OPC_UA_Endpoints() {
```

Comment:
I know it's not your change. We could fix though this to follow the coding conventions.

Reply:
@cristipogacean, are you referring to test names or something else ?